department-of-veterans-affairs/caseflow
# Disable and uncheck special issues if 'No special issues' is selected

Resolves #13995

### Description
Unchecks and disables all other special issues if "No special issues" is selected

### Acceptance Criteria
- [x] All other issues are unchecked and disabled if "No special issues" is selected

### Testing Plan
1. Log in as an attorney
1. Pick an ama case to review and select "Draft decision"
1. Select any special issues other then no special issues
1. Click continue
1. Click back
1. Ensure the ones you selected are preserved
1. Select "No special issues"
1. Ensure all special issues are unchecked and you cannot click them
1. Click continue
1. Click back
1. Ensure "No special issues is still clicked and everything is disabled.
1. Hit cancel and confirm
1. Select "Draft decision"
1. Confirm no special issues is preserved
1. Complete checkout with the dev tools open to confirm request to special issues only shows no_special_issues is true
1. Log in as the judge this was sent to
1. Confirm no special issues was preserved
1. Repeat all of the above
1. Repeat all of the above for a legacy appeal

### User Facing Changes

 BEFORE|AFTER
 ---|---
![Screen Shot 2020-05-18 at 4 43 43 PM](https://user-images.githubusercontent.com/45575454/82257834-c1f3aa80-9926-11ea-8fb4-4725ff5c9a47.png)|![Screen Shot 2020-05-18 at 5 11 33 PM](https://user-images.githubusercontent.com/45575454/82260179-c752f400-992a-11ea-98f3-ac3813afcd45.png)
